source/_integrations/here_travel_time.markdown

```diff
@@ -2,10 +2,11 @@
 title: HERE Travel Time
 description: Instructions on how to add HERE travel time to Home Assistant.
 ha_category:
-  - Sensor
   - Transport
 ha_iot_class: Cloud Polling
 ha_release: '0.100'
+ha_config_flow: true
+ha_quality_scale: silver
```

Comment:
Hmm I don't see that the manifest currently has this set?

Reply:
I opened a PR for it and think it even is gold: https://github.com/home-assistant/core/pull/72308